sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -672,6 +679,29 @@ public override void Dispose()
                     OperationTimeout = _operationTimeout
                 };
 
+                if (isSessionReceiver && sessionId == null)
+                {
+                    // Subtract a random amount up to 100ms from the operation timeout as the jitter when attempting to open next available session link.
+                    // This prevents excessive resource usage when using high amounts of concurrency and accepting the next available session. Without the jitter,
+                    // we can get many timeout exceptions occurring at the exact same time which leads to high CPU usage and thread starvation,
+                    // particularly when using the session processor.
+                    // Take the min of 1% of the total timeout and the BaseJitter amount so that we don't end up subtracting more than 1% of the total timeout.
+                    var jitterBase = Math.Min(_operationTimeout.TotalMilliseconds / 100, OpenReceiveLinkBaseJitterMilliseconds);
+
+                    // We set the operation timeout on the properties not only to include the jitter, but also because the server will otherwise
+                    // restrict the maximum timeout to 1 minute and 5 seconds, regardless of the client timeout. We only do this for accepting next available
+                    // session as this is the only long-polling scenario.
+                    linkSettings.Properties = new Fields
+                    {
+                        {
+                            AmqpClientConstants.TimeoutName,
+                            (uint)_operationTimeout
```

Comment:
So the service timeout is always shorter than the client?

Reply:
Yes, the service already subtracts 20ms off of their hardcoded 1 minute and 5 second timeout. This adds an additional buffer.